frontend/src/app/main/data/workspace/variants.cljs

```diff
@@ -563,9 +570,9 @@
        vec))
 
 (defn combine-as-variants
-  ([]
-   (combine-as-variants nil {}))
-  ([selected {:keys [page-id]}]
+  ([options]
```

Comment:
Making the first argument as optional looks confusing and is unexpected on code. 

The best approach for handle this, is have two events: one high-level that takes no selected argument and deduces it from the "context"/"state", and then the lower-level event which receives the collected context (the "selected" and others) and executes the main logic.